docs/program/apis/disconnect.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Session Connectivity: Network Disconnection during a Session"
+linkTitle: "Network Disconnection"
+weight: 20
+type: "docs"
+description: "How Viam handles losing connection to the internet during a robot session."
+tags: ["client", "sdk", "viam-server", "networking", "apis", "robot api", "session"]
+---
+
+A guide to what happens when your robot loses its network connection during a session.
+
+### How does `viam-server` work if you lose connectivity during a session?
+
+If you lose connectivity, `viam-server` will by default timeout all resource and end this robot's [*session*](/program/apis/sessions/), therefore stopping the flow of power to all actuating resources.
+
+This is because the robot cannot access Viam's APIs, including the robot API, without a connection to the cloud.
+
+To disable the default behavior here and manage resource timeout and reconfiguration over a networking session yourself, follow [these instructions](/program/apis/sessions/) to [disable the default behavior](/program/apis/sessions/#disable-default-session-management) of session management, then utilize [Viam's SDKs](/program/) in your code to make calls to [the session management API](https://pkg.go.dev/go.viam.com/rdk/session#hdr-API).
+If you do this, and configure the primary and sub-parts of your robot to cache all tokens so that authentication to Viam's cloud is not required for operation-related communication over the local network or directly through `viam-server`, you can operate your robot consistently without the need for a network connection.
+However, the low-network approach is not recommended if you want to utilize any of Viam's [services](/services/), like the vision service, motion planning, or the frame system.
+
+### How do Viam's SDKs work if you lose connectivity during a session?
+
+As a client of your robot, the Python or TypeScript SDKs will have their connection to the robot stopped when the robot loses connectivity, if the session is ended properly.
+If you wish to implement your own client SDK, note that client implementation of session management is necessary to make this work.
+The Go Client SDK is a little bit different: as the session management API is implemented through the Go Client, the session can be managed from within this client, allowing for more flexibility in timing disconnection.
+
+### How will your robot's primary parts and sub parts work if you lose connectivity during a session?
+
+Resources on the robot will no longer be connected and signaling to each other if you lose connectivity and the session is ended.
```

Comment:
This is not true. The CLIENT (aka "you") may be disconnected from the resource, but that doesn't mean resources that depend on other resources are disconnected from each other at all.

Reply:
Ok tried to make better, I was confused-- LMK if still inaccurate @Otterverse